src/Tkmm/Resources/locales.json

```
@@ -1650,6 +1650,50 @@
       "zh_CN": "",
       "zh_TW": ""
     },
+    "SetupWizard_MissingKeys_Title": {
+      "ar_SA": "",
+      "de_DE": "",
+      "el_GR": "",
+      "en_US": "Keys are missing",
+      "es_ES": "",
+      "fr_FR": "",
+      "he_IL": "",
+      "it_IT": "",
+      "ja_JP": "",
+      "ko_KR": "",
+      "no_NO": "",
+      "pl_PL": "",
+      "pt_BR": "",
+      "ru_RU": "",
+      "sv_SE": "",
+      "th_TH": "",
+      "tr_TR": "",
+      "uk_UA": "",
+      "zh_CN": "",
+      "zh_TW": ""
+    },
+    "SetupWizard_MissingKeys_Content": {
+      "ar_SA": "",
+      "de_DE": "",
+      "el_GR": "",
+      "en_US": "Keys are missing, please reboot to Hekate and inject the Lockpick_RCM payload in order to dump them.",
```

This could be worded better.

```suggestion
      "en_US": "The production keys (prod.keys) and/or title keys (title.keys) are missing, please reboot to Hekate and inject the Lockpick_RCM payload to dump them.",
```